src/content/fr/2019/caching.md

```diff
@@ -412,15 +412,15 @@ Si l'on compare les ressources du domaine principal et celles des tierces partie
     </tr>
     <tr>
      <td>Bureau</td>
-     <td><p style="text-align: right">70.7&nbsp;%</p></td>
-     <td><p style="text-align: right">47.9&nbsp;%</p></td>
-     <td><p style="text-align: right">59.2&nbsp;%</p></td>
+     <td><p class="numeric">70.7&nbsp;%</p></td>
+     <td><p class="numeric">47.9&nbsp;%</p></td>
+     <td><p class="numeric">59.2&nbsp;%</p></td>
```

Comment:
```suggestion
     <td><p class="numeric">70.7%</p></td>
     <td><p class="numeric">47.9%</p></td>
     <td><p class="numeric">59.2%</p></td>
```

Reply:
This is intentional as noted in the [translators guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Translators'-Guide). In french they write "70 %" (with a space) while in English we use "70%" (without a space). We use a non-breaking space to prevent orphaning a number from the percent sign when they fall on a line break.